crates/ruff/src/args.rs

```diff
@@ -515,101 +554,184 @@ impl From<&LogLevelArgs> for LogLevel {
     }
 }
 
+/// Struct to encapsulate all configuration-related arguments passed via the CLI.
+///
+/// Three kinds of configuration arguments can be provided via the CLI;
+/// each kind is represented by a separate field on this struct.
```

Comment:
I would also say this is implied by the datastructure and can easily become outdated, I wouldn't include it.